types/nconf/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for nconf
```

Comment:
This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `// Type definitions for nconf 0.10`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!

Reply:
Fixed, thanks!  I didn't see anything else in tslint that stood out as egregious, though I may revisit why there are so many disabled rules in a future PR.